clients/cpp-console/configs/micConfig.json

```diff
@@ -0,0 +1,16 @@
+{
+    "micArrayType": "Linear",
+    "numberOfMicrophones": 2,
+    "micCoord": [
+        {
+            "xCoord": 0,
+            "yCoord": 0,
+            "zCoord": 0
+        },
+        {
+            "xCoord": 0,
+            "yCoord": 0,
+            "zCoord": 0
+        }
+    ]
+}
```

Comment:
Can you give a more realistic example here? (where mic1 does not equal mic2 coordinates). Something from a mic device.

Reply:
See here https://docs.microsoft.com/en-us/azure/cognitive-services/speech-service/how-to-devices-microphone-array-configuration for some examples

Reply:
done and added link in readme to the docs page about mic configurations